sw/device/silicon_creator/lib/drivers/flash_ctrl_unittest.cc

```diff
@@ -528,9 +601,9 @@ INSTANTIATE_TEST_SUITE_P(AllCases, FlashCtrlCfgSetTest,
                                  .cfg = {.scrambling = kMultiBitBool8True,
                                          .ecc = kMultiBitBool8False,
                                          .he = kMultiBitBool8False},
-                                 .info_read_val = 0x7f,
+                                 .info_read_val = 0x0,
```

Comment:
nit: I can update these while removing the translation (as you mentioned in the PR description) but I was curious why you set some of the `*_read_val`s to `0x0`. These were non-zero to test that the functions preserve the values of the bits that they are not supposed to modify. Was it because these values don't make sense anymore because of the change in the layouts of the registers?

Reply:
o sorry, thank you for pointing this out. I did this temporarily because I got lazy and didn't want to write another helper function that would take in the individual bits and convert them to all mubi false.
I also wasn't sure if there was a better way to test them in general.